### DISCUSSION TITLE
Errors with table identification in PDF (false positives)

### DISCUSSION THREAD

**Discussion by kcbz** [jsvine/pdfplumber#1227] [Ask for help with specific PDFs]
I believe this is a bug, but I have a PDF of only text and for every page of the PDF, pdfplumber believes that the entire content of each page belongs to a table. For my purposes I need tables and text to identify components of the document correctly and separately (I know text captures table data but I created a work around for this). I've never really seen this reverse case where pdfplumber thinks there are tables when there are not any.

I have tried playing with the table_settings, but this didn't fix the issue, I also tried using the debug_tablefinder() and it seems to confirm that it thinks the contents of every page is a table.

I attached the PDF below:
[emeryville-ca-TITLE_10_TIDELANDS.pdf](https://github.com/user-attachments/files/17712456/emeryville-ca-TITLE_10_TIDELANDS.pdf)

**Comment by jsvine** [comment]
Hi @kcbz, this doesn't appear to be a bug, but rather a tricky aspect of this PDF, which is that it contains some not-visible rects:

```python
import pdfplumber
pdf = pdfplumber.open("emeryville-ca-TITLE_10_TIDELANDS.pdf")
page = pdf.pages[0]
im = page.to_image()
im.draw_rects(page.rects)
```

![image](https://github.com/user-attachments/assets/75491e5f-3b4c-4ebd-b066-e0c546bd3f94)

In cases like these, I recommend examining the `page.rects` objects directly. In this case, they're not appearing visible because they're white — i.e., `'non_stroking_color': (1, 1, 1)` — on a white background.

To ignore those rects, you can use the `page.filter(...)` method (example in a few discussions and issues, e.g., https://github.com/jsvine/pdfplumber/discussions/1219#discussioncomment-11210228).

**Reply by kcbz** [reply]
ok thanks!
